src/test/java/uk/nhs/tis/trainee/notifications/event/GmcListenerTest.java

```diff
@@ -235,7 +235,10 @@ void shouldIncludeCcedToInRejectTraineeTemplate() throws MessagingException {
         sentTemplateVars.get(TIS_TRIGGER_FIELD), is(TIS_TRIGGER));
     assertThat("Unexpected template tis trigger detail field.",
         sentTemplateVars.get(TIS_TRIGGER_DETAIL_FIELD), is(TIS_TRIGGER_DETAIL));
+    String ccField = (String) sentTemplateVars.get(CC_OF_FIELD);
     assertThat("Unexpected template cc of field.",
-        sentTemplateVars.get(CC_OF_FIELD), is("lo@1.com; lo@2.com"));
+        ccField.contains("lo@1.com"), is(true));
+    assertThat("Unexpected template cc of field.",
+        ccField.contains("lo@2.com"), is(true));
```

Comment:
A better assertion would be to split the string so we can check it doesn't contain a mystery third value.
Possibly consider actually changing out the actual HashSet used to a LinkedHashSet for consistent ordering of recipients.

Reply:
True, I was being a bit hasty to get this thing out the door.

For simplicity I've gone with a list, although in retrospect that does not obviously guarantee uniqueness from a method signature point of view...

Reply:
A TreeSet would be a good option if uniqueness is a concern.

Reply:
Well, the code makes it unique (the set-based aggregation is applied in the containing iteration through the local office contacts). I might as well make the signature reflect that though.

Reply:
A good example of more haste, less speed. Cleaned up now I think.